starfish/image/_registration/_apply_transform/_base.py

```diff
@@ -10,6 +10,10 @@
 
 
 class ApplyTransform(PipelineComponent):
+    """
+    ApplyTransform exposes methods to align image data by transforming and re-sampling input data
```

Comment:
transforming _or_ re-sampling?

translation shouldn't re-sample.

Reply:
Translation does re-sample, since we do it at sub-pixel shifts.